images/base-glibc-debian-bash/Dockerfile

```diff
@@ -26,6 +26,15 @@ RUN apt-get update -qq \
     apt-get remove --yes \
       locales \
     && \
+    # On Debian 10 (and 11) libgl1-mesa-glx pulls in libgl1-mesa-dri (which in
+    # turn has more heavy-weight dependencies). We leave these out of the image
+    # (by manually removing it from "Depends:" list) like we do with Debian 9.
+    sed -i \
+      '/^Depends:/ s/, libgl1-mesa-dri\>//g' \
+      /var/lib/dpkg/status \
+    && \
+    apt-get autoremove --yes \
+    && \
```

Comment:
@bgruening, in https://github.com/bioconda/bioconda-containers/pull/6#discussion_r595944644 I wrote
> But, well I have no idea if there are any notable things that would break by this.

Well, I'm still no expert here, but judging from https://salsa.debian.org/xorg-team/lib/mesa/-/commit/97cac862b360ac1891c76f39d61008fbd8edbc89#9c96da0e9f91d7d8937b69b524702c106258f0d1_6_7
>   * control: Bump libgl1-mesa-dri to libglx-mesa0 Depends, it's needed
>     for swrast used on some tests.

I'd guess it's not too bad. No idea, if subsequent changes added more reliance on this, but from just "swrast and some tests" doesn't sound too crazy.

Reply:
Yeah, that looks also good to me. A risk that we can take. We will see soon if that was wrong :)